docs/Configuration.md

```diff
@@ -0,0 +1,22 @@
+# Configuration
+
+The GUI can be configured by changing the editing the file `assets/app.config.json`.
+
+The supported configuration items are the following.
+
+* `"apiEndpoint"`: The URL to use to contact the API, default `"/api"`.
+* `"defaultLanguage"`: The default language of the GUI, defined as a two letters
+  code, default `"en"`. Accepted values: `"da"`, `"en"`, `"fi"`, `"fr"`,
+  "`nb"`, `"sv"`.
+* `"enabledLanguages"`: An array of the languages enabled in the GUI, default
+  `[ "da", "en", "fi", "fr", "nb", "sv" ]`.
+* `"contactAddress"`: The contact email address displayed in the footer, default
+  `"contact@zonemaster.net"`.
+* `"logo`": The URL to the image displayed in the navigation bar, default
+  `"assets/images/zonemaster_logo_2021_color.png"`.
```

Comment:
The Zonemaster logo is part of the GUI. I see no reason why it should be possible to change that with a simple configuration. If you want to change that, do a recompilation.

Reply:
I am not sure, it is common for other software to allow for easy customization of the GUI. I don't see the point for *forcing* a recompilation when the option is already available, documented or not.

Reply:
You mean that you could just replace the logo file. That is correct, so OK, keep it.